pipeline/transformer/transformer_calm/src/test/scala/uk/ac/wellcome/platform/transformer/calm/CalmTransformerWorkerTest.scala

```diff
@@ -0,0 +1,71 @@
+package uk.ac.wellcome.platform.transformer.calm
+
+import java.time.Instant
+
+import akka.stream.scaladsl.{Sink, Source}
+import org.scalatest.{FunSpec, Matchers}
+import uk.ac.wellcome.bigmessaging.memory.MemoryBigMessageSender
+import uk.ac.wellcome.models.work.internal.TransformedBaseWork
+import uk.ac.wellcome.models.Implicits._
+import uk.ac.wellcome.storage.store.memory.{MemoryStore, MemoryVersionedStore}
+import com.amazonaws.services.sqs.model.Message
+import uk.ac.wellcome.akka.fixtures.Akka
+import uk.ac.wellcome.fixtures.TestWith
+import uk.ac.wellcome.platform.transformer.calm.models.CalmRecord
+import uk.ac.wellcome.messaging.sns.NotificationMessage
+import uk.ac.wellcome.storage.Version
+import uk.ac.wellcome.storage.maxima.Maxima
+import uk.ac.wellcome.storage.maxima.memory.MemoryMaxima
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+class CalmTransformerWorkerTest extends FunSpec with Matchers with Akka {
+  it("Runs") {
```

Comment:
Can we get a better description for this test?

Reply:
`"Never succeeds running a flow with an invalid source"`